frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -181,85 +169,56 @@ export function PrintTestDeckScreen(): JSX.Element {
       ? undefined
       : getPrecinctById({ election, precinctId: precinctIds[precinctIndex] });
 
-  if (precinctIds.length > 0) {
-    return (
-      <React.Fragment>
-        {precinctIndex !== undefined && currentPrecinct && (
-          <Modal
-            centerContent
-            content={
-              <Loading as="p">
-                Printing Test Deck
-                {precinctIds.length > 1
-                  ? ` (${precinctIndex + 1} of ${precinctIds.length})`
-                  : ''}
-                : {currentPrecinct.name}
-              </Loading>
-            }
-          />
-        )}
-        <NavigationScreen>
-          <Prose>
-            <h1>{pageTitle}</h1>
-            <p>
-              <strong>Election:</strong> {election.title}
-              <br />
-              <strong>Precinct:</strong> {precinctName}
-            </p>
-            <p>
-              <Button onPress={startPrint} primary>
-                Print Test Deck
-              </Button>
-            </p>
-            <p>
-              <LinkButton small to={routerPaths.testDecks}>
-                Back to Test Deck list
-              </LinkButton>
-            </p>
-          </Prose>
-        </NavigationScreen>
-        {precinctIndex !== undefined && (
-          <TestDeckBallotsMemoized
-            election={election}
-            electionHash={electionHash}
-            precinctId={precinctIds[precinctIndex]}
-            onAllRendered={onAllRendered}
-          />
-        )}
-      </React.Fragment>
-    );
-  }
-
   return (
-    <NavigationScreen>
-      <Prose>
-        <h1>{pageTitle}</h1>
+    <React.Fragment>
+      {precinctIndex !== undefined && currentPrecinct && (
+        <Modal
+          centerContent
+          content={
+            <Loading as="p">
+              Printing Test Deck
+              {precinctIds.length > 1
+                ? ` (${precinctIndex + 1} of ${precinctIds.length})`
+                : ''}
+              : {currentPrecinct.name}
+            </Loading>
+          }
+        />
+      )}
+      <NavigationScreen>
+        <Prose>
+          <h1>{pageTitle}</h1>
+          <p>
+            Select desired precinct for <strong>{election.title}</strong>.
+          </p>
+        </Prose>
         <p>
-          Select desired precinct for <strong>{election.title}</strong>.
+          <Button onPress={() => startPrint('all')} fullWidth>
+            <strong>All Precincts</strong>
+          </Button>
         </p>
-      </Prose>
-      <p>
-        <LinkButton to={routerPaths.testDeck({ precinctId: 'all' })} fullWidth>
```

Comment:
Given the removal of the sub-page, I _think_ we can now remove the `testDeck` route from `routerPaths` (correct me if I'm wrong and it's still used somewhere else!)

https://github.com/votingworks/vxsuite/blob/387bd5b02fa3c3444f90d8e44ba6683e6eb1f278/frontends/election-manager/src/router_paths.ts#L52-L53

Reply:
Ah, yes! Good catch, thank you. I'll push a change and remove the router path.